iothub/device/tests/CustomType.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    internal class CustomType
+    {
+        [JsonProperty]
+        internal string stringAttri;
```

Comment:
Follow .NET casing standards, and use attributes to fix serialization casing.

```suggestion
        [JsonProperty("stringAtri")]
        internal string StringAttri;
```